lottie/src/main/java/com/airbnb/lottie/LottieDrawable.java

```diff
@@ -794,14 +795,52 @@ public void playAnimation() {
       }
     }
     if (!animationsEnabled()) {
-      setFrame((int) (getSpeed() < 0 ? getMinFrame() : getMaxFrame()));
+      Marker markerForAnimationsDisabled = getMarkerForAnimationsDisabled();
+      if (markerForAnimationsDisabled != null) {
+        setFrame((int) markerForAnimationsDisabled.startFrame);
+      } else {
+        setFrame((int) (getSpeed() < 0 ? getMinFrame() : getMaxFrame()));
+      }
       animator.endAnimation();
       if (!isVisible()) {
         onVisibleAction = OnVisibleAction.NONE;
       }
     }
   }
 
+  /**
+   * The marker to use if "reduced motion" is enabled.
+   * Supported marker names are case insensitive, and include:
+   *   - reduced motion
+   *   - reducedMotion
+   *   - reduced_motion
+   *   - reduced-motion
+   */
+  private static final List<String> ALLOWED_REDUCED_MOTION_MARKERS = Arrays.asList(
```

Comment:
Could you move this to the top of LottieDrawable (after invalidateSelfOnMainThread)?

Reply:
Moved. thanks for the help.